packages/ddp-client/common/livedata_connection.js

```diff
@@ -534,7 +534,7 @@ export class Connection {
    * @memberOf Meteor
    * @importFromPackage meteor
    * @alias Meteor.call
-   * @summary Invokes a method passing any number of arguments.
+   * @summary Invokes a sync method passing any number of arguments.
```

Comment:
Should we update it to say something like "Invokes a method with a sync stub, passing any number of arguments." instead?

Reply:
Sure, that's fine!